velox/exec/tests/RowContainerTest.cpp

```diff
@@ -49,6 +49,9 @@ class RowContainerTest : public exec::test::RowContainerTestBase {
     testExtractColumnOutOfOrder(container, rows, column, expected, 0);
     testExtractColumnOutOfOrder(
         container, rows, column, expected, rows.size() / 3);
+
+    testRepeatedRowNumbers(container, rows, column, expected);
```

Comment:
nit: Any particular reason these methods use different naming scheme than similar method testExtractColumnOutOfOrder ? Would it make sense to rename testExtractColumnOutOfOrder to testOutOfOrderRowNumbers?